src/pages/[lang]/tutorial.astro

```diff
@@ -0,0 +1,9 @@
+---
+import languages from '../../i18n/languages';
+
+export const getStaticPaths = () => Object.keys(languages).map((lang) => ({ params: { lang } }));
+
+const { lang } = Astro.params;
+---
+
+<meta http-equiv="refresh" content={`0;url=/${lang}/tutorial/0-introduction/`} />
```

Comment:
Will this show up in the sitemap?

Reply:
Excellent question. The answer is: yes!

https://deploy-preview-1976--astro-docs-2.netlify.app/sitemap-0.xml

We can filter sitemap URLs I believe. Let me look at that — we should probably exclude our other similar redirects too.

Reply:
OK, question back time: does it matter if it is in sitemap?

Reply:
Its usually best to keep redirects out of the sitemap, but that can be tackled in a separate PR :+1:

Its really not a huge deal in general, I think it's pretty common for redirects to linger in a sitemap for a while (or forever on some sites)

Reply:
OK — pushed a fix for the low-hanging fruit here. Our `[lang]/something.astro` redirects like the one added in this PR are easy to filter out. Other dynamic redirects in the root of `src/pages/` will be trickier and can be tackled later I guess!

Reply:
Engaged morning brain and implemented a much better filtering set-up so now the sitemap shouldn’t include any of our redirect routes!